pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```diff
@@ -187,4 +187,8 @@ public static Boolean isValueUnique(@NonNull DBElement dbElement) {
         });
         return (Boolean) results.resultValue;
     }
+
+    public int getParentIdAsInt() {
```

Comment:
Instead of going full bore and changing various fields from String -> Int/Long, I added this utility method to make some incremental progress.